modules/yahooAdvertisingBidAdapter.js

```diff
@@ -6,9 +6,9 @@ import { Renderer } from '../src/Renderer.js';
 import {hasPurpose1Consent} from '../src/utils/gpdr.js';
 
 const INTEGRATION_METHOD = 'prebid.js';
-const BIDDER_CODE = 'yahoossp';
+const BIDDER_CODE = 'yahooAdvertising';
+const BIDDER_ALIASES = ['yahoossp']
```

Comment:
maybe yahoo as well?

You are welcome to change your bidder code and alias the old adapter, however, changing the filename is breaking because of the build instructions. https://github.com/prebid/Prebid.js/blob/a35cda0896d9223ac2f0f6c90e201593b30851e3/README.md?plain=1#L170

You'll have to wait for prebid 9 to do that. We just shipped Prebid 8 last week.